inspirehep/modules/workflows/templates/inspire_workflows/index.html

```diff
@@ -77,13 +74,34 @@ <h3 class="text-center header-text">
 
       <holding-pen-dashboard-item
         template="{{url_for('static', filename='js/inspire_workflows_ui/templates/dashboard_stat_block.html')}}"
-        primary_filter_key="source"
-        primary_filter_value="arXiv"
+        filter_string="source=arXiv"
         secondary_filter="status"
         section_title="arxiv harvests">
       </holding-pen-dashboard-item>
 
     </div>
+
+    <div class="col-md-3 col-sm-6">
+
+      <holding-pen-dashboard-item
+        template="{{url_for('static', filename='js/inspire_workflows_ui/templates/dashboard_stat_block.html')}}"
+        filter_string="source=Alpha experiment"
```

Comment:
There are nearly 20 sources. It might then not be too feasible to then present all of them separately.
E.g. many of them are actually simply harvesting of theses from different websites, so maybe they can be presented in one place.

Also the Alpha experiment is going to publish probably a handful of publications and that's it. So the box is going to have a very brief life.

How difficult will it be to amend these boxes? Could it be worth pinging catalogers to ask what they need? @eamonnmag?